src/lint/lint-ts.ts

```diff
@@ -1,41 +1,84 @@
-import { Linter, LintResult } from "tslint";
+import { Linter, LintResult, Configuration } from "tslint";
+import { resolve } from "path";
+import { CommandModule, Argv, argv } from "yargs";
 
 import { Logger } from "../utils/logger";

```

Comment:
Actually good spot, though i can do somehing like  import { Logger } from "../utils"; ie omit the index as it will resolved automatically